.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
You should add this into your global ignore configuration, this configuration is not specific to the projet but to your development environment.